packages/ember-toucan-form/.eslintignore

```diff
@@ -6,3 +6,7 @@
 
 # misc
 /coverage/
+
+# until we have linting for gts set up
```

Comment:
Looking at https://github.com/CrowdStrike/ember-toucan-core/pull/124/commits/c3ac7e0263b88051c6bdc354b3d835c88c50f1d0

There doesn't seem to be any prettier changes? Just deletions? Just to confirm.

Reply:
Yeah! Exactly!  Fresh add-on so not much here yet